packages/dashboard/src/components/task-summary-panel.tsx

```diff
@@ -24,6 +23,45 @@ export const getActorFromStatus = (status: string) => {
   return status.match(/\[[A-Za-z]([a-zA-Z0-9\/]){3,}\]+/gi);
 };
 
+export const formatStatus = (status: string) => {
+  return status.split('|');
+};
+
+export const getStateLabel = (state: number): string => {
+  switch (state) {
+    case RomiCore.TaskSummary.STATE_QUEUED:
+      return 'QUEUED';
+    case RomiCore.TaskSummary.STATE_ACTIVE:
+      return 'ACTIVE';
+    case RomiCore.TaskSummary.STATE_COMPLETED:
+      return 'COMPLETED';
+    case RomiCore.TaskSummary.STATE_FAILED:
+      return 'FAILED';
+    default:
+      return 'UNKNOWN';
+  }
+};
+
+interface TaskSummaryPanelInfoProps {
+  task: RomiCore.TaskSummary;
```

Comment:
The only prop is the list of task, we should be able to migrate this to `react-components` as well.

While we are at it, we should also move the save/restore functions out into a HOC or hook. Try and see if you can make it more generic, what we need is a way to
  * save the current state
  * restore the saved state
  * merge the restored state with the current state
  * clear the saved state
I think it should be possible to write a generic hook to handle that.

This component should only handle rendering a list of tasks, the task panel in `dashboard` will then use this component, along with the HOC/hook to handle the saving/clearing logic.